docs/governance/how-is-etherlink-governed.md

```diff
@@ -13,14 +13,29 @@ Like Tezos's governance process, Etherlink's governance process promotes transpa
 
 For information about governance proposals and updates, see https://governance.etherlink.com.
 
+For the addresses of the contracts that manage governance, see [How do I participate in governance?](/governance/how-do-i-participate-in-governance).
+
 ## Kernel governance
 
 The Etherlink kernel governance process is a streamlined version of the [Tezos governance and self-amendment process](https://docs.tezos.com/architecture/governance).
 It consists of three periods: a Proposal period and a Promotion period, which are supervised by Etherlink's kernel governance contract, and a Cooldown period, which is enforced by the Etherlink kernel itself.
 
+The lengths of these periods are stored in the [kernel governance contract](https://better-call.dev/mainnet/KT1H5pCmFuhAwRExzNNrPQFKpunJx1yEVa6J).
+This table shows the period lengths as of the Bifröst Etherlink update and the Tezos Quebec protocol:
+
+Period | Length | Approximate time
+--- | --- | ---
+Proposal | 30720 layer 1 blocks | About 3 days
+Promotion | 30720 layer 1 blocks | About 3 days
```

Comment:
This are the proposed values with Calypso. The current values are 

> We have a period of 20480 (~45.5 hours).

from @pecornilleau on Slack

Reply:
Corrected to 20480.